task_manager/api.py

```diff
@@ -66,46 +63,48 @@ def delete(self, task_id):
 
 @api.route('/user/create', methods=['GET', 'POST'])
 class CreateUser(Resource):
-	''' CREATE USER '''
 	def get(self):
-		return {'Show':'Form'}
+		# return {'Show':'Form'}
+		return model_get_create_user()
 
 	def post(self):
-		return {'create':'user'}
+		# return {'create':'user'}
+		return model_post_create_user()
 
 
 @api.route('/user/')
 @api.route('/user/<int:user_id>')
 class FetchUser(Resource):
-	''' FETCH USER '''
 	def get(self, user_id = -1):
-		return {'User': user_id}
+		# return {'User': user_id}
+		return model_fetch_user(user_id)
 
 
 @api.route('/user/edit/<int:user_id>', methods=['GET', 'POST'])
 class UpdateUser(Resource):
-	''' UPDATE USER '''
 	def get(self):
-		return {'Show':'Form'}
+		# return {'Show':'Form'}
+		return model_get_update_user(user_id)
```

Comment:
F405: 'user_id' may be undefined, or defined from star imports: routes.tasks, routes.users  <!-- Automatically posted by Lintly -->